### DISCUSSION TITLE
Unable to run tutorial last solution: `ModuleNotFoundError: No module named 'trame_vtk.modules.vtk.serializers'`

### DISCUSSION THREAD

**Discussion by kerim371** [Kitware/trame#575] [Q&A] [answered]
Hi,

I just started using TRAME with the [tutorial](https://github.com/Kitware/trame-tutorial)

The last solution (04_application) I cant run, the error is:
```
(venv) PS C:\Users\me\dev\trame-tutorial> python ./04_application/solution.py --port 1234
Traceback (most recent call last):
  File "C:\Users\me\dev\trame-tutorial\04_application\solution.py", line 7, in <module>
    from trame_vtk.modules.vtk.serializers import configure_serializer
ModuleNotFoundError: No module named 'trame_vtk.modules.vtk.serializers'
```

What `configure_serializer` should actually do?

Probably I should try another version of Trame?

**Comment by jourdain** [comment, marked as answer]
Just try `pip install trame-vtk -U`

**Comment by kerim371** [comment]
@jourdain thank tou, the problem was solved,

But I got another one:
TRAME app loads too long and can't start for already 2 minuts and I see a message in terminal:
```
wslink is not expecting text message:
> {"wslink":"1.0","id":"system:c0:0","method":"wslink.hello","args":[{"secret":"wslink-secret"}],"kwargs":{}}
```
![image](https://github.com/user-attachments/assets/21c9fe55-e608-4dea-af8a-2c7f1eb31fb6)

**Reply by jourdain** [reply]
Ok you have a mix of version between server and client.
Just choose a different port or clear your browser cache.

**Reply by kerim371** [reply]
thank you!! cool!
